scroll-animations-1/Overview.bs

```diff
@@ -736,67 +736,95 @@ spec: cssom-view-1; type: dfn;
 
 	A set of animation keyframes can be attached
 	in reference to [=named timeline ranges=]
-	by adjusting the duration of the animation.
+	by restricting the [=animation active phase=]
+	to the specified timeline range.
+	Delays ('animation-start-delay'/'animation-end-delay')
```

Comment:
Shouldn't these be:
```suggestion
	Delays ('animation-delay-start'/'animation-delay-end')
```
?

Reply:
Yes. :/ Switched to “(see 'animation-delay')”.